### PR TITLE
Convert Footer and ViewOnlyDecrypt to stateless functional components

### PR DESCRIPTION
**Pull request by kangarang**
Hello, I want to help out with this awesome project! I'm comfortable with React, and to get a better picture of how the app works, I'll move around and clean up some of the bloat. Let me know if there's anything in particular you'd like me to work on.

### CODE REVIEW

**Comment by tayvano** [comment]
Come join our slack and I'll get you in the internal channel :D 

https://myetherwallet.herokuapp.com/

**Comment by dternyak** [comment]
@kangarang Welcome! Thanks for contributing.

It looks like you have accidentally checked in some modifications to the package-lock.json, do you mind removing that?

**Comment by kangarang** [comment]
Hello, thanks! Yes, just reverted back.

File: common/components/WalletDecrypt/index.jsx [hunk 1]
```diff
@@ -29,19 +29,24 @@ const WALLETS = {
...
     lid: 'x_Trezor',
-    component: TrezorDecrypt
+    component: TrezorDecrypt,
+    initialParams: {}
   },
   'view-only': {
     lid: 'View with Address Only',
-    component: ViewOnlyDecrypt
+    component: ViewOnlyDecrypt,
+    initialParams: '',
```
Comment (dternyak) [inline common/components/WalletDecrypt/index.jsx]: initial params `''` here but `{}` everywhere else?

**Comment by kangarang** [comment]
Ah, I was a bit confused at first til I did some reading on flow and got a better idea of the wallet conventions like the ones in #103. Thanks!

File: common/components/Footer/index.jsx [hunk 2]
```diff
@@ -1,206 +1,206 @@
...
 import { bityReferralURL } from 'config/data';
 import './index.scss';
 
-export default class Footer extends Component {
-  render() {
-    const translators = getTranslators();
-    return (
-      <footer className="Footer" role="contentinfo" aria-label="footer">
-        <div className="container">
+const Footer = () => {
```
Comment (crptm) [inline common/components/Footer/index.jsx]: function Footer() { }

Has exact same length

**Comment by dternyak** [comment]
@kangarang Could you fix conflicts and remove the package-lock.json change? Then I'll merge.

Thanks!

**Comment by kangarang** [comment]
Closing this for now. Will reopen with much smaller and cleaner commits.

**Pull request closed without merging**